### PR TITLE
Type hints and docstrings

### PR DESCRIPTION
**Pull request by martinbaerwolff**
OP#652

### CODE REVIEW

**Review by randy-seng — APPROVED** [review]
@martinbaerwolff I fixed some problems along the way. Please check my most recent commmits. But from my end all good!

**Comment by martinbaerwolff** [comment]
Hi Randy, I had to make some small fixes to get everything to run from CLI. Could you have a look?

(We really have to implement a higher test coverage during the upcoming sprint - I see that very clear now :) )

File: OTVision/transform/transform.py [hunk 1]
```diff
@@ -21,51 +21,66 @@
...
 
     log.info("Start transformation from pixel to world coordinates")
     if debug:
-        log.setLevel("DEBUG")
-        log.debug("Debug mode on")
+        set_debug()
 
     if refpts_file:
+        refpts_file = get_files(
+            paths=[refpts_file], filetypes=CONFIG["FILETYPES"]["REFPTS"]
```
Comment (randy-seng) [inline OTVision/transform/transform.py:82]: What happens, if more than one refpts files can be found?
The assumption is that there is only one, right? But how do we determine the correct one when several could be found?

This is not that urgent for this PR and can be addressed later. Best to document it.

**Review by randy-seng — COMMENTED** [review]
Just one question that popped up. Other than that, all good!

File: OTVision/transform/transform.py (hunk 1 above)
Comment (martinbaerwolff) [reply OTVision/transform/transform.py:82]: I think if we pass only one file path to get files with only one possible filetype, only one refpts_file can be found. But yes, it is kind of ugly right now. We have to improve this and also the filetypes in the config

Comment (martinbaerwolff) [reply OTVision/transform/transform.py:82]: I noted this for later

**Review by randy-seng — APPROVED** [review]

**Pull request merged**